Ref/test/int/ref_integration_test.py

```diff
@@ -264,3 +266,9 @@ def test_active_logger_filter(self):
             self.api.assert_event_count(pred, actHI_events)
         finally:
             self.set_default_filters()
+
+    def test_seqgen(self):
+        """ Tests the seqgen code """
+        sequence = os.path.join(os.path.dirname(__file__), "test_seq.seq")
+        assert subprocess.run(["fprime-seqgen", "-d", self.dictionary, sequence, "/tmp/ref_test_int.bin"]).returncode == 0, "Failed to run fprime-seqgen"
```

Comment:
You may want to make this a temporary file/directory with a random name https://docs.python.org/3/library/tempfile.html

Makes the system slightly more resilient against unexpected state, like where the file already exists (like where the integration test previously crashed).

Reply:
tmpfile can create longer file paths and there is an issue with cmdSeq and long paths.  So I'll leave this for now.

Reply:
How does the length of the filepath on the host system affect seqgen? I can get that the remote filepath length will be limited, but the host path shouldn't be used anywhere should it?